tests/functional/pod_and_daemons/test_csiaddon_pod_security.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import logging
+
+from ocs_ci.ocs.resources.pod import get_pods_having_label
+from ocs_ci.ocs.resources.pod import Pod
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.constants import CSI_RBDPLUGIN_LABEL_419
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    green_squad,
+)
+
+log = logging
+
+
+@tier1
+@green_squad
+class TestCSIAddonPodSecurity:
+    """This class contain test to Validates CSI Addon pod enforces security
```

Comment:
This class contains tests to Validate if CSI Addon pod enforces security

Reply:
Done